### ISSUE TITLE
NameError: name '__version__' is not defined

### ISSUE THREAD

**Issue by dmalan** [ryanfox/sphinx-markdown-tables#10]
Hi, using Python 3.6.5 (installed with `pyenv`), afraid it looks like version 0.0.7 is erring with the below:

```
$ sphinx-build -M html "." "_build"
Running Sphinx v1.7.6

Exception occurred:
  File "/usr/local/lib/python3.6/dist-packages/sphinx_markdown_tables.py", line 8, in setup
    return {'version': __version__}
NameError: name '__version__' is not defined
The full traceback has been saved in /tmp/sphinx-err-99y19ebc.log, if you want to report the issue to the developers.

$ cat /tmp/sphinx-err-99y19ebc.log
# Sphinx version: 1.7.6
# Python version: 3.6.5 (CPython)
# Docutils version: 0.14 
# Jinja2 version: 2.10
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/sphinx/cmdline.py", line 303, in main
    args.warningiserror, args.tags, args.verbosity, args.jobs)
  File "/usr/local/lib/python3.6/dist-packages/sphinx/application.py", line 191, in __init__
    self.setup_extension(extension)
  File "/usr/local/lib/python3.6/dist-packages/sphinx/application.py", line 411, in setup_extension
    self.registry.load_extension(self, extname)
  File "/usr/local/lib/python3.6/dist-packages/sphinx/registry.py", line 326, in load_extension
    metadata = mod.setup(app)
  File "/usr/local/lib/python3.6/dist-packages/sphinx_markdown_tables.py", line 8, in setup
    return {'version': __version__}
NameError: name '__version__' is not defined
```

If helpful, too, it looks like `__version__.py` is getting installed into `/usr/local/lib/python3.6/dist-packages` itself, rather than in a package directory like `/usr/local/lib/python3.6/dist-packages/sphinx_markdown_tables`:

```
$ ls -l /usr/local/lib/python3.6/dist-packages
...
-rwxr-xr-x  1 root staff   1212 Jul 28 15:33 sphinx_markdown_tables.py
-rwxr-xr-x  1 root staff     23 Jul 28 15:33 __version__.py
...
```

**Comment by jkmacc-LANL** [comment]
FYI, I hard-coded `'0.0.7'` to replace the `__version__` variable on the offending line, just to see if this extension worked.  It did:smile:

**Comment by ryanfox** [comment]
Thanks for the catch. Should be fixed in v0.0.8, on pypi now.

**Comment by dmalan** [comment]
Working well now! Thank you!
